README.md

```diff
@@ -202,7 +202,7 @@ docker run --rm -v $(pwd):/src -w /src -e TF_VAR_key_vault_firewall_bypass_ip_ci
 ```
 
 On Windows Powershell:
-
+``
```

Comment:
Is this a typo ?

Reply:
Thanks @zioproto for review this pr! Would you please give another review? Thanks.